models/faq.js

```diff
@@ -0,0 +1,24 @@
+const mongoose = require('mongoose');
+const Schema = mongoose.Schema;
+
+const faqSchema = new Schema(
+  {
+    faqCategoryName: {
```

Comment:
If we have categories for faqs, I think faqCategory should have its collection, and be linked to faq collection.